src/layerstack.stanza

```diff
@@ -278,7 +286,7 @@ public defstruct LayerStack :
   description:Maybe<String>
 
   layers:Vector<LayerSpec> with: (
-    updater => sub-layers
+    setter => sub-layers
```

Comment:
You renamed it but didn't convert it back to `updater` here. You have left it as a `setter`.

Reply:
Oops!